.circleci/config.yml

```diff
@@ -1375,3 +1406,50 @@ workflows:
       equal: [bump, << pipeline.parameters.action >>]
     jobs:
       - release-train
+
+  # to trigger tests manually, log into circleCI, select the project, a branch, and then click "Trigger Pipeline"
+  # and add a parameter called "run-manual-tests"
+  manual-test-workflow:
+    when:
+      and:
+        - not:
+            equal: [scheduled_pipeline, << pipeline.trigger_source >>]
+        - not: << pipeline.parameters.generate_snapshots >>
+        - not: << pipeline.parameters.generate_revenuecatui_snapshots >>
+        - or:
+            - matches: 
+                pattern: "^release/.*$"
+                value: << pipeline.git.branch >>
+            - equal:
+                - "manual_tests"
+                - << pipeline.trigger_source >>
```

Comment:
doesn't trigger for scheduled pipelines or snapshot pipelines, and will be triggered by release branches and manual trigger